src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -20,6 +20,7 @@
         "ext-xml": "*",
         "symfony/cache": "^4.3|^5.0",
         "symfony/config": "^4.2|^5.0",
+        "symfony/debug": "^4.4|^5.0",
```

Comment:
Shouldn't it be in the `require-dev` section instead?

Reply:
That wouldn't fix the issue then as the dev dependencies are only applied if they are part of the root project's `composer.json` file.

Reply:
We should also update the FB recipe (`public/index.php`) and check by `class_exists(Debug::class)` too? inline with the `bin/console` recipe.

Reply:
So this should be part of the https://github.com/symfony/skeleton/blob/master/composer.json instead? in `require-dev`?

Reply:
Maybe with https://github.com/symfony/symfony/pull/32270#discussion_r298742507 we solve the issue.

Reply:
The main problem are the existing applications. Especially because you get a fully white page (500 error) since the `ErrorCatcher` error handler and exception hander were never registered.

I guess Flex can update the front controller index.php file but is it automatic ? Checking if the class exists would make the error disappear but that would change the default current behavior. That would mean that every project would need to require `symfony/debug` manually. That would be better but I guess that implies a lot of communication.

AFAIK, Flex cannot add a dependency to the composer.json file right?

Reply:
Right, It wouldn't solve the problem for actual projects updating to 4.4, but new projects. Meanwhile, let's fix it for the new ones.